source/common/protobuf/utility.h

```diff
@@ -182,6 +182,15 @@ class MessageUtil {
    */
   static std::string getJsonStringFromMessage(const Protobuf::Message& message);
 
+  /**
+   * Extract JSON as string from a google.protobuf.Message.
+   * @param message message of type type.googleapis.com/google.protobuf.Message.
+   * @param pretty_print whether the returned JSON should be formatted.
+   * @return std::string of formatted JSON object.
+   */
+  static std::string getJsonStringFromMessage(const Protobuf::Message& message,
+                                              const bool pretty_print);
```

Comment:
nit: I would just add `pretty_print` to the other definition and default it to false, then just remove this version.